index.js

```diff
@@ -12,7 +12,7 @@ app.use(bodyParser());
 //serve Manticore webpage if enabled
 //Must build it first by running 'npm run build-webpage'
```

Comment:
Is this still true? It looks like the build step was removed from the dockerfile.